src/components/BlogCard/BlogCard.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types";
+import { getAuthorById, getCategoryById } from "../../utils/helperData";
+import "./BlogCard.scss";
+
+const BlogCard = (blog) => {
+  const { bannerImage, categoryId, title, authorId, createdAt } = blog.blog;
+  const category = getCategoryById(categoryId);
+  const author = getAuthorById(authorId);
+  return (
+    <div className="blog-card">
+      <img src={bannerImage} alt={title} className="blog-card__banner" />
+      <div className="blog-card__category-badge">
+        {category ? category.name : "Unknown Category"}
+      </div>
+      <h3 className="blog-card__blog-title">{title}</h3>
+      <div className="blog-card__author-info">
+        <img
+          src={author ? author.profileImage : ""}
```

Comment:
Is there any possibility where a blog would not have an author?
const defaultPlaceholder = "../assets/default-placeholder.svg"

```suggestion
          src={author.profileImage ? author.profileImage : defaultPlaceholder}
```

Reply:
Checking because initially an author has only first name, last name and username.